### PR TITLE
Android E2E Multiple Emulators Fixes 

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Misc fixes to `walletconnect.spec.js` and changes to the `run_e2e.sh` to better support multiple emulators / unblock CI.

### Other changes

N/A

### Tested

Tested locally and in CI.

### How others should test

This does not need to be tested by QA. Reviewers should check the history of runs of this branch in iOS and Android E2E tests.

### Related issues

N/A

### Backwards compatibility

Yes

### CODE REVIEW

File: .github/workflows/e2e-android.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ jobs:
           export PATH=$ANDROID_SDK_ROOT/platform-tools:$PATH
           export PATH=$ANDROID_SDK_ROOT/emulator:$PATH
           cd packages/mobile
-          yarn run test:e2e:android -w 3
+          yarn run test:e2e:android -w 1
```
Comment (kathaypacific) [inline .github/workflows/e2e-android.yml:68]: is this back to 1 on purpose? 🥲

File: packages/mobile/e2e/src/WalletConnect.spec.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ describe('Given Wallet Connect', () => {
     await quickOnboarding()
   })
 
+  // Testing to see if walletConnectV1 is the issue
```
Comment (kathaypacific) [inline packages/mobile/e2e/src/WalletConnect.spec.js:10]: is this comment necessary?

File: packages/mobile/e2e/src/usecases/WalletConnectV1.js [hunk 3]
```diff
@@ -149,7 +149,7 @@ export default WalletConnect = () => {
     let result = walletConnector.signTransaction(tx)
     await waitFor(element(by.text('Sign a Celo TX')))
       .toBeVisible()
-      .withTimeout(10 * 1000)
+      .withTimeout(15 * 1000)
```
Comment (kathaypacific) [inline packages/mobile/e2e/src/usecases/WalletConnectV1.js:152]: i wonder if it's worthwhile to make a wrapper for this line and the one above, like `toBeVisibleWithTimeout()` and inside that function you could control the timeout...makes it easier to maintain the timeout values?

File: packages/mobile/scripts/run_e2e.sh [hunk 4]
```diff
@@ -190,12 +189,6 @@ if [ $PLATFORM = "android" ]; then
   # Run Detox Tests
   runTest
 
-  # Close active emulators
```
Comment (kathaypacific) [inline packages/mobile/scripts/run_e2e.sh:193]: is detox closing the emulators after the tests by default? or are we leaving the emulators running until the next run?

File: .github/workflows/e2e-android.yml (hunk 1 above)
Comment (MuckT) [reply .github/workflows/e2e-android.yml]: Was attempting to see if that was the issue, but I've bumped it back up to 3.

File: packages/mobile/e2e/src/usecases/WalletConnectV1.js (hunk 3 above)
Comment (MuckT) [reply packages/mobile/e2e/src/usecases/WalletConnectV1.js:152]: The Detox Maintainers have indicated they want to move away form `waitFor` syntax, but I am unsure on the priority in their roadmap: https://github.com/wix/Detox/issues/2190. Can wrap this in the utilities in another PR.

File: packages/mobile/scripts/run_e2e.sh (hunk 4 above)
Comment (MuckT) [reply packages/mobile/scripts/run_e2e.sh:193]: Added this back in, as it does appear as if it was closing the emulators in CI after the run.

File: packages/mobile/e2e/src/WalletConnect.spec.js (hunk 2 above)
Comment (MuckT) [reply packages/mobile/e2e/src/WalletConnect.spec.js]: Removed in https://github.com/valora-inc/wallet/pull/1837/commits/7609f7d602b2cf6e67e6b41843c51bd3d993a984!

File: packages/mobile/e2e/src/usecases/WalletConnectV1.js (hunk 3 above)
Comment (gnardini) [reply packages/mobile/e2e/src/usecases/WalletConnectV1.js:152]: +1 to moving it to an utils or somewhere where we can put a default timeout so it's easier to change. If they do end up changing the `waitFor` syntax we would only need to change it in one place (probably).

**Review by gnardini — APPROVED** [review]
Not sure I fully understand what's going on with all the `disableSynchronization`s and their order with other assertions but it looks like it's unblocking CI so that's nice :D

**Comment by ValoraQA** [comment]
Hey @MuckT since "How others should test has n/a, " hence we are skipping this PR

**Pull request merged**